public/app/plugins/panel/table/renderer.ts

```diff
@@ -53,7 +53,7 @@ export class TableRenderer {
         return style.colors[i];
       }
     }
-    return _.first(style.colors);
+    return style.colors[0];
```

Comment:
I made this because I notice that `style.colors` is a dictionary (and not an array).  
So, if the `value` is a number less than the minimum threshold, the value/cell/row isn't colored.